packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -256,14 +256,13 @@ $btn-sizeXSmall-iconHeight             : 12px;
 $btn-sizeXSmall-iconSpacing            : 2px;
 $btn-sizeXSmall-iconSideSpacing        : $btn-sizeXSmall-iconSpacing + spacing();
 
-$btn-icon-sizeLarge-buttonSize         : 56px; // button--icon is a sircle so width and height can use one var
+$btn-icon-iconSize                     : 18px; // at the moment all the icon buttons except large have the same icon size
```

Comment:
Just checking Figma – is this true? I can see [different icon sizes depending on the button size](https://www.figma.com/file/R2rBfzJP0hG0MZorq6FLZ1/%F0%9F%9A%A7-PIE-2.-0-Global---Light?node-id=991%3A5466).